compose.yml

```diff
@@ -8,25 +7,88 @@ services:
     networks:
       - cdp-tenant
 
-  your-frontend:
-    build: ./
+  # your-frontend:
+  #   build: ./
```

Comment:
@DrogoNevets  - We're going to need this block for our pr checker to work.  Just flagging it here as a reminder, I know this is WIP.